transforms/language/doc_quality/python/README.md

```diff
@@ -1,13 +1,21 @@
 # Document Quality Transform 
+
 Please see the set of
 [transform project conventions](../../../README.md#transform-project-conventions)
 for details on general project conventions, transform configuration,
 testing and IDE set up.
 
-## Summary 
-This transform will calculate and annotate several metrics related to document, which are usuful to see the quality of document. 
+## Description 
+This transform will calculate and annotate several metrics related to document, which are usuful to see the quality of document.
+Text is the type of data this transform operates on.
+
+### Input 
 
-In this transform, following metrics will be included:
+| input column name | data type | descrition |
+|-|-|-|
+| the one specified in _doc_content_column_ configuration | string | text whose quality will be calculated by this transform |
```

Comment:
User has to provide entire text of document as a string in the column specified in _doc_content_column_ configuration so that transform can calculate metrics of the text.

Reply:
Should we not call it content then instead of description in table header ?